runtime/router.go

```diff
@@ -53,21 +53,21 @@ type RouterEndpoint struct {
 // NewRouterEndpoint creates an endpoint with all the necessary data
 func NewRouterEndpoint(
 	gateway *Gateway,
-	endpointName string,
-	handlerName string,
+	endpointID string,
+	handlerID string,
 	handler HandlerFn,
 ) *RouterEndpoint {
 	logger := gateway.Logger.With(
-		zap.String("endpointID", endpointName),
-		zap.String("handlerID", handlerName),
+		zap.String("endpointID", endpointID),
+		zap.String("HandlerID", handlerID),
```

Comment:
please stop renaming stuff when it does not need to change thanks. `handlerID` not `HandlerID`

Reply:
Done.